File: Network/BreakLoadAndUpdate OC/EOCBreakLoadImage/ViewController.m
//
//  ViewController.m
//  EOCBreakLoadImage
//
//  Created by EOC on 2017/6/13.
//  Copyright © 2017年 EOC. All rights reserved.
//

#import "ViewController.h"
#import "BreakLoadVC.h"
#import "BreakLoadOptVC.h"
#import "EOCUpdateFileVCtr.h"
#import "SysLoadVC.h"


@interface ViewController () <UITableViewDelegate, UITableViewDataSource>{
    
 
}


@end


@implementation ViewController

- (void)viewDidLoad {
    
    [super viewDidLoad];
   
    NSString *docPath = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) lastObject];
    NSLog(@"%@", docPath);
    
  

}


- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section{
    
    
    return 4;
}

- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath{
    
    
    return 60;
}


- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath{
    
    
    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"cell"];
    if (!cell) {
        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:@"cell"];
    }
    if (indexPath.row == 0) {
        cell.textLabel.text = @"breadLoad";
    }
    if (indexPath.row == 1) {
        cell.textLabel.text = @"BreakLoadOptimize";
    }
    if (indexPath.row == 2) {
        cell.textLabel.text = @"SysLoad";
    }
    if (indexPath.row == 3) {
        cell.textLabel.text = @"update";
    }
    
    return cell;
    
}

- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
    
    if (indexPath.row == 0) {
        
        [self.navigationController pushViewController:[BreakLoadVC new] animated:YES];
    }
    if (indexPath.row == 1) {
        
        [self.navigationController pushViewController:[BreakLoadOptVC new] animated:YES];
    }
    if (indexPath.row == 2) {
        
        [self.navigationController pushViewController:[SysLoadVC new] animated:YES];
    }
    if (indexPath.row == 3) {
        
        EOCUpdateFileVCtr *eocBreadLoadVCtr = [EOCUpdateFileVCtr new];
        [self.navigationController pushViewController:eocBreadLoadVCtr animated:YES];
    }
    
}



@end

File: Network/BreakLoadAndUpdate/BreakLoadAndUpdate/ViewController.swift
//
//  ViewController.swift
//  BreakLoadAndUpdate
//
//  Created by Casey on 04/03/2019.
//  Copyright © 2019 nb. All rights reserved.
//

import UIKit

class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {

    override func viewDidLoad() {
        super.viewDidLoad()
        
    }
    
    
    //MARK: tableview delegate
    
    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return 4
    }
    
    
    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
        
        return 64
    }
    
    
    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        
        var cell = tableView.dequeueReusableCell(withIdentifier: "cell")
        if cell == nil {
            cell = UITableViewCell.init(style: .default, reuseIdentifier: "cell")
        }
        if indexPath.row == 0 {
            cell?.textLabel?.text = "breadLoad"
        }
        if indexPath.row == 1 {
            cell?.textLabel?.text = "BreakLoadOptimize"
        }
        if indexPath.row == 2 {
            cell?.textLabel?.text = "SysLoad"
        }
        if indexPath.row == 3 {
            cell?.textLabel?.text = "update"
        }
        return cell!
    }
    
    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
        tableView.deselectRow(at: indexPath, animated: true)
        
        
        if indexPath.row == 0 {
            
            self.navigationController?.pushViewController(BreakLoadViewCtr(), animated: true)
            
        }else if indexPath.row == 1 {
            
             self.navigationController?.pushViewController(BreakLoadOptimizeViewCtr(), animated: true)
            
        }else if indexPath.row == 2 {
            
            self.navigationController?.pushViewController(SysLoadViewCtr(), animated: true)
            
        }else if indexPath.row == 3{
            
            self.navigationController?.pushViewController(UpdateFileViewCtr(), animated: true)
        }
        
        
        
    }

    

}

